settings.js

```diff
@@ -92,6 +92,13 @@ window.spinnakerSettings = {
     snapshots: false,
     functions: functionsEnabled,
   },
+  filterRelations: {
```

Comment:
It is not required in the configs because we are generating the relationship between the filters in this, and we assume that the relationship is universal.

Reply:
Wouldn't config in `settings.js` require a rebuild/redploy of deck in order to update?

Configurable filters is something our users have been asking for as well, but different teams/apps will want to filter on different things.